tests/framework/utils/k8s_helper.go

```diff
@@ -64,6 +62,16 @@ const (
 	hostnameTestPrefix = "test-prefix-this-is-a-very-long-hostname-"
 )
 
+// GetKubectlCmd returns kubectl or oc if env var rook_test_openshift is
+// set to true
+func GetKubectlCmd() string {
```

Comment:
@leseb `GetCLICmd()` sounds better?

Reply:
Yes, it seems more accurate. Or maybe just `GetCmd()`?

Reply:
sound good!

Reply:
Done.